res/database/Default/user/jorgaeff.xml

```diff
@@ -2356,9 +2748,11 @@
 		<news id="news-jorgaeff-fanta-02" type="2" thread_id="news-jorgaeff-fanta" creator="8936" created_by="jorgaeff">
 			<title>
 				<de>Heribert Fanta ist Easiophobiker</de>
+				<en>Heribert Fanta is Easiophobic</en>
 			</title>
 			<description>
 				<de>Nur wenige Menschen leiden an Easiophobie - der Angst zu schreiben. Der deutsche Erfolgsautor Heribert Fanta ist einer von Ihnen. Er möchte sich künftig nur noch auf die Produktion von Film-Dokumentationen konzentrieren.</de>
+				<en>Few people suffer from easiophobia - the fear of writing. The successful German author Heribert Fanta is one of them. In the future he would like to concentrate only on the production of film documentaries anymore.</en>
```

Comment:
remove "anymore"

Reply:
it's actually not wrong, just a very bold stylistic choice.

Moving 'only' to the end of the sentence instead.